tests/cephadm/test_check_non_exisiting_legacy_services.py

```diff
@@ -0,0 +1,43 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Verify if the cephadm shell tries to infer config from
+    a dummy mon created.
+    Verifies bz:2080242
+
+    Args:
+        ceph_cluster: Ceph cluster object
+        **kw :     Key/value pairs of configuration information to be used in the test.
+
+    Returns:
+        int - 0 when the execution is successful else 1 (for failure).
+    """
+
+    config = kw.get("config")
+
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+
+    for node in ceph_cluster.get_nodes():
+        # Create a dummy mon
+        hostname = node.hostname
+        dummy_mon = f"sudo touch /var/lib//ceph/mon/ceph-{hostname}"
+        _, err = node.exec_command(cmd=f"{dummy_mon}")
+        if err:
+            log.error("Failed to create dummy mon")
+            return 1
+
+        # Try cephadm shell command to make sure the above dummy mon doesn't interfere
+        # Checking cluster health
+        status = cephadm.cluster.check_health(
+            rhbuild=config.get("rhbuild"), client=cephadm.installer
+        )
+        if status:
+            log.info("The dummy mon doesn't crash the cephadm shell")
```

Comment:
crash is not at all meaningful over here,refer https://bugzilla.redhat.com/show_bug.cgi?id=2080242#c20
basically, it should not get into cephadm shell from the dummy mon node
`As expected not able to access cephadm shell from dummy mon node` hope it would be more meaningful.

Reply:
Ack. Noted, the log message will be reformatted